app/javascript/components/supplemental_application/sections/ConfirmedHouseholdIncome.js

```diff
@@ -44,13 +44,12 @@ const ConfirmedHouseholdIncome = ({ amis, amiCharts, formApi }) => {
           </FormGrid.Group>
         </FormGrid.Item>
         <FormGrid.Item>
-          <FormGrid.Group label='Household Annual Income'>
-            <Text field='annual_income'
+          <FormGrid.Group label='Confirmed Household Annual Income'>
```

Comment:
From the AC it looks like this should be updated to Confirmed Annual income so the field title doesn't wrap